develop/best_practices/Best_practices.md

```diff
@@ -0,0 +1,9 @@
+---
+uid: Best_practices
+---
+
+# Best practices
+
+In this section:
+
+- [Creating unit tests in protocols](xref:Creating_unit_tests_in_protocols)
```

Comment:
```suggestion
- [Creating unit tests](xref:Creating_unit_tests_in_protocols)
```
I would keep the title more general "Creating unit tests" as the explanation is also applicable to e.g. Automation scripts or solutions in general (the text does explain it for a protocol but it's very similar for Automation scripts).